### DISCUSSION TITLE
jvm/gc

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#287] [Announcements]
# jvm/gc

本篇内容我们从头到尾讲了一遍 JVM 的垃圾回收机制，包括垃圾回收的概念、垃圾判断算法、垃圾收集算法、Stop The World、新生代和老年代等等。

https://javabetter.cn/jvm/gc.html

**Comment by chaofengdev** [comment]
我在图里看到了二哥对自己的吐槽。说自己是个沙雕。

**Reply by chaofengdev** [reply]
我宣布这篇是我看的最清晰的一篇。

**Comment by ZLDXR** [comment]
引用计数算法的英文写错了，应为ReferenceCounting
